requirements.txt

```diff
@@ -7,4 +7,9 @@ pymongo==2.8
 Flask==0.10.1
```

Comment:
Why docker s required? You can use it with virtualenv/etc as usual

if requirements.txt will be read into into setup.py it will work without docker as well... and of course it will work if we exctract initial set of requirements from setup.py some way